change/gytis-drop-Node-12-14_2023-02-01-15-55.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
These change files have different file name format. Can you make sure that `beachball` will handle them correctly?

Reply:
```
beachball check
Checking for changes against "origin/master"
fetching latest from remotes "origin/master"
Your local repository already has change files for these packages:
  @itwin/presentation-opentelemetry
  @itwin/presentation-testing
Validating no private package among package dependencies
 OK!

No change files are needed
```